.github/workflows/wheels.yml

```diff
@@ -54,17 +54,15 @@ jobs:
           # Pass MAX_JOBS=4 because, at time of writing, the VM "only" has 32GB
           # of RAM and OOMs while building if we give it the default number of
           # workers (2 * NUM_CPUs).
-          #
-          # Sadly, I couldn't make TRITON_BUILD_WITH_CLANG_LLD=1 work.  The
-          # manylinux image has a relatively recent gcc (v10, released 2020),
-          # but its clang is ancient, v3.4, released in 2014 (!).  I tried
-          # installing the prebuilt clang 10 binary distributed by LLVM, and I
-          # quickly ran into Linux DLL hell.  I give up, for now.  Perhaps
-          # manylinux_x_y will save us; I didn't try.
-          export CIBW_ENVIRONMENT="MAX_JOBS=4 TRITON_WHEEL_NAME=triton-nightly TRITON_WHEEL_VERSION_SUFFIX=-$LATEST_DATE"
-          export CIBW_MANYLINUX_X86_64_IMAGE="quay.io/pypa/manylinux2014_x86_64:latest"
-          #export CIBW_MANYLINUX_PYPY_X86_64_IMAGE="quay.io/pypa/manylinux2014_x86_64:latest"
-          export CIBW_BEFORE_BUILD="pip install cmake;"
```

Comment:
This is not needed because the dependency is covered by toml file's build dependencies.